graphql-app/pages/index.js

```diff
@@ -0,0 +1,108 @@
+import Head from "next/head";
+import axios from "axios";
+import { useState } from "react";
+
+export default function Home() {
+  const [countryName, setCountryName] = useState(null);
+  const [response, setResponse] = useState(null);
+
+  const getSearchResults = async () => {
+    try {
+      const res = await axios.get("api/countries/", {
+        params: { countryName },
+      });
+      const { data } = res;
+      setResponse(data.data.countries.edges);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return (
+    <div className="flex flex-col relative bg-background font-raleway items-center min-h-screen ">
+      <Head>
+        <title>RapidAPI Devrel Example - GraphQL App</title>
+        <link rel="icon" href="/favicon.ico" />
+        <link
```

Comment:
Ideally, this should be present in **app.jsx**.

Reply:
Agreed. I left it there because I knew it was a single page app. Moved now.